README.md

```diff
@@ -0,0 +1,163 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="cat-black-face.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio: finish mobile version</b></h3>
```

Comment:
- You currently do not have a profesional README file, however this is an important requirement in future proyects you will build here at Microverse. ("Why is it important?") 👍
- Please make sure you make the changes to the README.md file correctly, it is very important that your project is properly named. 😃